Files/Filesystem/ListedItem.cs

```diff
@@ -204,11 +204,14 @@ private void Img_ImageOpened(object sender, Windows.UI.Xaml.RoutedEventArgs e)
 
                 if (image.PixelWidth > 0)
                 {
-                    LoadFileIcon = true;
-                    PlaceholderDefaultIcon = null;
-                    NeedsPlaceholderGlyph = false;
-                    LoadDefaultIcon = false;
-                    LoadWebShortcutGlyph = false;
+                    Common.Extensions.IgnoreExceptions(() =>
+                    {
+                        LoadFileIcon = true;
+                        PlaceholderDefaultIcon = null;
+                        NeedsPlaceholderGlyph = false;
+                        LoadDefaultIcon = false;
+                        LoadWebShortcutGlyph = false;
+                    }, App.Logger); // 2009482836u
```

Comment:
Why would it fail?

Reply:
Good question, I do not know for sure.The referenced stacktrace has always been there but we had a try-catch around it.